blocks/api/serializer.js

```diff
@@ -50,20 +50,28 @@ export function getSaveContent( blockType, attributes ) {
 	}
 
 	// Adding a generic classname
-	const addClassnameToElement = ( element ) => {
-		if ( ! element || ! isObject( element ) || ! className ) {
+	const addGenericAttributes = ( element ) => {
```

Comment:
@mtias I think you'd mentioned these sorts of attributes (`class`, `id`) being "advanced". Maybe a more appropriate adjective here?